arduino-ide-extension/src/browser/boards/boards-auto-installer.ts

```diff
@@ -106,7 +106,7 @@ export class BoardsAutoInstaller implements FrontendApplicationContribution {
                 });
                 return;
               }
-              if (answer) {
+              if (answer === 'Install Manually') {
```

Comment:
this change is suspicious and not test related. Did we mess up with commits?

Reply:
No, I changed this on purpose. In the current implementation, we are checking if the answer is `'Yes'`: if so, we install the library; in any other case, we open the library manager. I didn't like that, because if we'd add another answer like `'Nah, I don't want to'`, we don't want to open it.